src/mixins/cell/cell.js

```diff
@@ -46,10 +44,16 @@ export default {
 
     computed: {
         cellClasses () {
+            const isExactMatch = this.row._exactMatch &&
+              this.column.property === (this.row._meta && this.row._meta.filterProperty);
             return Object.assign(
                 this.cssProcessor.process(null, this.columnIndex, this.column),
                 this.cssProcessor.process(this.rowIndex + 1, this.columnIndex, this.row, this.column),
-                this.cssProcessor.processFixed(this.row._classes, this.columnIndex, this.row, this.column)
+                this.cssProcessor.processFixed(this.row._classes, this.columnIndex, this.row, this.column),
+                isExactMatch ? {
+                    'bg-yellow-500': true,
+                    'cell-exact-match': true,
```

Comment:
I guess the match highlighting should be configurable via the `classes` property similar to how other fixed selectors are, e.g.:
\- `selected`: Style rows selected by end users.

Please also mention this in the README styling section.